### PR TITLE
Updates the remote libraries (jquery, bootstrap to latest minor versi…

### PR DESCRIPTION
**Pull request by mikeckennedy**
Updates the remote libraries (jquery, bootstrap to latest minor versions).

Bootstrap 3.0.3 -> 3.3.7
Jquery 1.10.2 -> 1.12.4
respond.js 1.3.0 -> 1.4.2

### CODE REVIEW

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/layout.jinja2 [hunk 1]
```diff
@@ -11,15 +11,15 @@
...
+    <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
 
     <!-- Custom styles for this scaffold -->
     <link href="{{ '{{request.static_url(' }}'{{cookiecutter.repo_name}}:static/theme.css'){{ '}}' }}" rel="stylesheet">
 
     <!-- HTML5 shim and Respond.js IE8 support of HTML5 elements and media queries -->
     <!--[if lt IE 9]>
       <script src="//oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js" integrity="sha384-0s5Pv64cNZJieYFkXYOTId2HMA2Lfb6q2nAcx2n0RTLUnCAoTTsS0nKEO27XyKcY" crossorigin="anonymous"></script>
-      <script src="//oss.maxcdn.com/libs/respond.js/1.3.0/respond.min.js" integrity="sha384-f1r2UzjsxZ9T4V1f2zBO/evUqSEOpeaUUZcMTz1Up63bl4ruYnFYeM+BxI4NhyI0" crossorigin="anonymous"></script>
+      <script src="//oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
```
Comment (mmerickel) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/layout.jinja2:22]: This line removes integrity?

**Comment by mmerickel** [comment]
Similar issue, please reopen against the master branch.

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/layout.jinja2 (hunk 1 above)
Comment (mikeckennedy) [reply {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/layout.jinja2:22]: I couldn't find the new SHA. I'd be happy to put it in if you know where it is.

**Comment by mikeckennedy** [comment]
@mmerickel Done, reopened against master. Thanks.

**Pull request closed without merging**